daemon/src/oracle.rs

```diff
@@ -195,7 +196,10 @@ impl Actor {
     ) -> Result<()> {
         tracing::info!("Fetched new attestation for {}", id);
 
-        let _ = self.attestation_channel.send(attestation).await;
+        self.attestation_channel
+            .send(attestation)
+            .log_failure()
+            .await?;
```

Comment:
To preserve semantics, we should do `let _ = ` here.

Reply:
Do we want to preserve the semantics though? We weren't taking advantage of returning a `Result` before. I can put in a separate commit if you prefer.

Reply:
I think in this first transition yes. We can then re-evaluate, where we fail for what reason.

Reply:
I've tried to follow your advice and preserve semantics all across the board. But I may have slipped, so do keep an eye out for mistakes like this one in your next review.